Update FAQ - Adding rules QA.md (#731)
SEKOIA-IO/documentation: docs/xdr/FAQ.md

# tail -n 1 /tmp/nginx-output.log
<30>1 2021-01-13T14:52:06.934860+01:00 ote unbound - LOG [SEKOIA@53288 intake_key="jOK5bMVXz5Iz7gfogQDbCcC7l7S2IrOs5"]  [596451:0] info: 127.0.0.1 intake.sekoia.io. A IN
```

# Rules

> ## In a rule, what is the meaning of successfully compiled ?

Reviewer: ```suggestion
## In a rule, what does "successfully compiled" mean?
```

> Meaning that the rule is correct and ongoing

Reviewer: ```suggestion
A rule that is successfully compiled means that it is correct and ongoing.
```

> To know the information of the rule update, it is `updated at`

Reviewer: ```suggestion
To access the rule's last update, you can search for the label `updated at` in the rule details panel. 

```

> ## Timestamp

Reviewer: ```suggestion
## What is the timestamp based on?
```

> Based on event timestamp (created on the machine) and not on ingestion timestamp in SEKOIA.IO

Reviewer: ```suggestion
The event timestamp is based on the event timestamp (when it's created in the machine), not on the ingestion timestamp in SEKOIA.IO. 
```
